src/components/TaskForm/TaskForm.js

```diff
@@ -0,0 +1,47 @@
+import { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import ACTIONS from '../../store/actions/actions';
+import TaskInput from '../TaskInput/TaskInput';
+
+import styles from './TaskForm.module.scss';
+import IconButton from '../IconButton/IconButton';
+
+export default function TaskForm({ tasks }) {
+  const [inputValue, setInputValue] = useState('');
```

Comment:
You could pass `handleSubmit` and `handleSubmitError` props to decouple from redux. You will have all the redux logic within app.jsx, that is really good for this task.

Reply:
I've done something bit diffrent, but in the spirit of your comment, I believe.